clients/common/src/components/JeevesContext.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import axios, { AxiosInstance } from 'axios';
+
+export interface IJeevesContext {
+  jeeves: AxiosInstance;
+}
+
+const jeevesAxiosInstance = axios.create({
+  baseURL:
+    process.env.NODE_ENV !== 'production'
+      ? 'http://localhost:80/api/v1'
```

Comment:
Make this configurable in a `.env` file. Different computer (i.e. windows) don't use localhost